collect-user-data

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```

Comment:
So what's the general consensus on node vs. bash? I've noticed some of these scripts are node and some are bash. This bash script integrates with a node one etc.

I tend toward node simply because I know it better than bash. I have written a few scripts on and off but honestly, nothing even as complicated as the one I'm commenting on. If I have something more complicated than automating some shell commands I've always dropped into another language, like python / js / clojure / whatever.

Interested in your perspective @mandric .